lms/templates/courseware/courseware-chromeless.html

```diff
@@ -139,6 +139,14 @@
 
         lastHeight = newHeight;
         lastWidth = newWidth;
+
+        // Within the learning microfrontend the iframe resizes to match the
+        // height of this document and it should never scroll. It does scroll
+        // ocassionally when javascript is used to focus elements on the page
+        // before the parent iframe has been resized to match the content
+        // height. This window.scrollTo is an attempt to keep the content at the
+        // top of the page. See TNL-7094
+        window.scrollTo(0, 0);
```

Comment:
Hmm - don't we want to do this only when we're loading the page?  I could see it potentially being a bad user experience to go to the top whenever they resize their browser window.  Can we add a flag or something that only does this the first time?

Reply:
Chatted in person. We're good!